.github/workflows/main.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
         fail-fast: false
         matrix:
-          runner: ["ubuntu-latest", "windows-latest", "macos-latest"]
+          runner: ["ubuntu-24.04", "windows-latest", "macos-latest"]
```

Comment:
I prefer to keep `ubuntu-latest`. I want the pipeline to fail should there ever be an issue with newer OS version. Also, it makes more sense to have all three OS' to be the same.

Reply:
Yes, this creates an asymmetry between ubuntu-22.04 and ubuntu-latest. However, specifying the Ubuntu version avoids build warnings like [this one](https://github.com/geekq/terraform-provider-kustomization/actions/runs/12198208222): "ubuntu-latest pipelines will use ubuntu-24.04 soon," announcing GitHub breaking changes.

Since we've confirmed ubuntu-22.04 works, it will still function after GitHub switches to 24.04 in December–January. I'll revert this commit: warnings will reappear, then disappear in 1-2 months.